src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/versionMatcher/versionObjectDpkg.hpp

```diff
@@ -0,0 +1,158 @@
+/*
+ * Wazuh Vulnerability scanner - Database Feed Manager
+ * Copyright (C) 2015, Wazuh Inc.
+ * December 06, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _VERSION_OBJECT_DPKG_HPP
+#define _VERSION_OBJECT_DPKG_HPP
+
+#include "iVersionObjectInterface.hpp"
+#include <iostream>
+#include <memory>
+#include <regex>
+#include <string>
+
+/**
+ * @brief dpkgVersion data struct.
+ *
+ */
+struct dpkgVersion
```

Comment:
Following the nomenclature of the other version objects, this struct should be called "Dpkg"

Reply:
Done